vim/ftplugin/fsharp.vim

```diff
@@ -88,8 +88,11 @@ EOF
         " closing the scratch window after leaving insert mode
         " is common practice
         au BufWritePre  *.fs,*.fsi,*fsx call fsharpbinding#python#OnBufWritePre() 
-        au TextChanged  *.fs,*.fsi,*fsx call fsharpbinding#python#OnTextChanged()
-        au TextChangedI *.fs,*.fsi,*fsx call fsharpbinding#python#OnTextChangedI()
+        if version > 703
+            " these events new in Vim 7.4
+            au TextChanged  *.fs,*.fsi,*fsx call fsharpbinding#python#OnTextChanged()
+            au TextChangedI *.fs,*.fsi,*fsx call fsharpbinding#python#OnTextChangedI()
+        endif
         au CursorHold   *.fs,*.fsi,*fsx call fsharpbinding#python#OnCursorHold()
```

Comment:
Hi - sorry about the delay in replying.

It is a bi strange - the OnBufWritePre event handle should send a parse request that is later waited on in the CurrentErrors function (blocking) until it completes. This should just work but obviously it doesn't for some reason. It might be better to change CurrentErrors to use fsharpbinding.errors() instead of fsharpbinding.errors_current() for 7.3 as this will cause it to send a parse request synchonously.